src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -66,6 +67,7 @@
     private $paginationPageParameterName;
     private $clientItemsPerPage;
     private $itemsPerPageParameterName;
+    private $apiGateway = false;
```

Comment:
Can't we do it without introducing a state?

Reply:
I tried to, but final code would have been much more complex